build-tools/packages/build-cli/src/codeCoverage/getCoverageMetrics.ts

```diff
@@ -0,0 +1,114 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import JSZip from "jszip";
+import { Parser } from "xml2js";
+import type { CommandLogger } from "../logging.js";
+
+/**
+ * The type for the coverage report, containing the line coverage and branch coverage(in percentage)
+ */
+export interface CoverageMetric {
+	lineCoverage: number;
+	branchCoverage: number;
+}
+
+interface TXmlCoverageReportSchema {
+	coverage: {
+		packages: TXmlCoverageReportSchemaForPackage[];
+	};
+}
+
+interface TXmlCoverageReportSchemaForPackage {
```

Comment:
nit: is there a significance to the "T" prefix?

Reply:
That was for type. But I think we are supposed to name it like that for templete types, not here. Removed it.